From patchwork Thu Apr 22 05:53:55 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Stephen Rothwell <sfr@canb.auug.org.au>
X-Patchwork-Id: 12217695
Return-Path: <kvm-owner@kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-10.8 required=3.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,HEADER_FROM_DIFFERENT_DOMAINS,INCLUDES_PATCH,
	MAILING_LIST_MULTI,SPF_HELO_NONE,SPF_PASS autolearn=unavailable
	autolearn_force=no version=3.4.0
Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
	by smtp.lore.kernel.org (Postfix) with ESMTP id 58BE3C433ED
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 05:54:08 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 2A36061425
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 05:54:08 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S234775AbhDVFyl (ORCPT <rfc822;kvm@archiver.kernel.org>);
        Thu, 22 Apr 2021 01:54:41 -0400
Received: from ozlabs.org ([203.11.71.1]:49913 "EHLO ozlabs.org"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S234806AbhDVFyh (ORCPT <rfc822;kvm@vger.kernel.org>);
        Thu, 22 Apr 2021 01:54:37 -0400
Received: from authenticated.ozlabs.org (localhost [127.0.0.1])
        (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
         key-exchange ECDHE (P-256) server-signature RSA-PSS (4096 bits)
 server-digest SHA256)
        (No client certificate requested)
        by mail.ozlabs.org (Postfix) with ESMTPSA id 4FQmnc44mMz9sTD;
        Thu, 22 Apr 2021 15:53:56 +1000 (AEST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=canb.auug.org.au;
        s=201702; t=1619070837;
        bh=BGZPYQkkrhZBKQnH/8AnrujpEbYer6hDxMUZS72H6e4=;
        h=Date:From:To:Cc:Subject:From;
        b=f8Sdg2jpgOZm3ETvhaBqhzh2fGNRPvOH7HcJ3kGq4wesKkELDZfgncJMVkZgLt+eD
         vS0ipQpTq9oSNkn3RMTcafzsQcnniCKxHwdtIQtEj+/4RFPLLCgJczSW7p2fDBsFUX
         zMiAhsPM8BG9G6NZTpW/CmfvxtUQVlaYHJHakw73VlEP1PMBc5C5QYPtLAoxHpft5T
         yYlGniklQBz66onN7adQBYUATthJgSSF3ZK6pZb5FQdkrtx8jvGEne7lf5omuF3rEZ
         kdD9hno/3KeaB1M9/Zosbijzwyn7rFr7AeVjnVkqUfFZgPKeV/ProH9Ddy9mwQE00m
         ljcErt3QFNm8g==
Date: Thu, 22 Apr 2021 15:53:55 +1000
From: Stephen Rothwell <sfr@canb.auug.org.au>
To: Tejun Heo <tj@kernel.org>, Paolo Bonzini <pbonzini@redhat.com>,
        KVM <kvm@vger.kernel.org>
Cc: Linux Kernel Mailing List <linux-kernel@vger.kernel.org>,
        Linux Next Mailing List <linux-next@vger.kernel.org>,
        Sean Christopherson <seanjc@google.com>,
        Vipin Sharma <vipinsh@google.com>
Subject: linux-next: manual merge of the cgroup tree with the kvm tree
Message-ID: <20210422155355.471c7751@canb.auug.org.au>
MIME-Version: 1.0
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Hi all,

Today's linux-next merge of the cgroup tree got conflicts in:

  arch/x86/kvm/svm/sev.c

between commit:

  9fa1521daafb ("KVM: SVM: Do not set sev->es_active until KVM_SEV_ES_INIT completes")

from the kvm tree and commit:

  7aef27f0b2a8 ("svm/sev: Register SEV and SEV-ES ASIDs to the misc controller")

from the cgroup tree.

I fixed it up (see below) and can carry the fix as necessary. This
is now fixed as far as linux-next is concerned, but any non trivial
conflicts should be mentioned to your upstream maintainer when your tree
is submitted for merging.  You may also want to consider cooperating
with the maintainer of the conflicting tree to minimise any particularly
complex conflicts.

diff --cc arch/x86/kvm/svm/sev.c
index 63923fa0b172,214eefb20414..000000000000
--- a/arch/x86/kvm/svm/sev.c
